src/reliability.jl

```diff
@@ -0,0 +1,78 @@
+using LinearAlgebra: isposdef
+
+struct Reliability{T <: Real}
+    alpha::T
+    dropped::Vector{Pair{Int, T}}
```

Comment:
The first parameter here should be `Any` not `Int`
```suggestion
    dropped::Vector{Pair{Any, T}}
```
I think.

This would allow passing names to items. The `Int` we have now is not useful (it is just a position in a vector so it is redundant).

@storopoli - I would wait with working on this on @nalimilan to comment about the API and passing item names here.

@nalimilan - I believe that the computational part in this PR is finished. We just need to decide on the way we handle the item names.

Reply:
I suggest we only store a `Vector{T}` of values for now. Names can be stored in a separate field if we add support for them later. Anyway it's more practical to store values separately from names than having to access the fields in a pair.